apps/DisplayCluster/resources/MasterContentWindow.qml

```diff
@@ -27,6 +27,26 @@ BaseContentWindow {
             displaygroup.focus(contentwindow.id)
     }
 
+    Item {
```

Comment:
Would not work with the
```
visible: contentwindow.state === ContentWindow.SELECTED
```
state, but the focus statue could be attached to that. I have to test if it still works then.